glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Multimedia 
++ #### *Packet* is a formatted unit of data transmitted over network 
++ #### *Frame* is a basic data unit used by media coding formats
++ #### (media)*Track* is equivalent to a single audio or video 
++ #### Web protocols:
+  + #### *UDP*(User Datagram Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connectionless communication
+  + #### *TCP*(Transmission Control Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connection-oriented communication
+  + #### *RTP*(Real-time Transport Protocol) is a network protocol for delivering audio and video over IP networks
+  + #### *HTTP*(Hypertext Transfer Protocol) is a protocol for fetching data from a server by a client
+  + #### (HTTP) *Long Polling* is a technique of keeping an open connection after client's request for as long as new data is not available. This is more efficient than naive repeated polling by a client until new data is received. 
+  + #### *WebRTC*(Web Real-Time Communication) is a free and open-source project providing web browsers and mobile applications with real-time communication (RTC)
+    + #### *signalling* in WebRTC is a process of discovery and establishing connection between two devices
```

Comment:
signalling is also responsible for controlling the connection and terminating it